File: output/ChatGPT/codenet/Python/C++/s043174379.cpp
#include <iostream>
using namespace std;

int main() {
    int H, W;
    cin >> H >> W;
    int h, w;
    cin >> h >> w;
    int ans = H * W - (h * W + w * H - h * w);
    cout << ans << endl;

    return 0;
}

File: output/ChatGPT/codenet/Python/Java/s043174379.java
import java.util.Scanner;

public class s043174379 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int H, W, h, w, ans;
        
        H = scanner.nextInt();
        W = scanner.nextInt();
        h = scanner.nextInt();
        w = scanner.nextInt();
        
        ans = H * W - (h * W + w * H - h * w);
        
        System.out.println(ans);
    }
}